Sources/Opus/Opus.Decoder.swift

```diff
@@ -5,9 +5,10 @@ public extension Opus {
 	class Decoder {
 		let format: AVAudioFormat
 		let decoder: OpaquePointer
+		let customFrameSize: Int32?
 
-		// TODO: throw an error if format is unsupported
 		public init(format: AVAudioFormat, application _: Application = .audio) throws {
+			customFrameSize = nil
```

Comment:
If this is made non-optional, this can be `customFrameSize = 0`, with `customMode = nil`.

Reply:
And then you need 2 new vars; customMode and customFrameSize.
Using an optional for customFrameSize means you don't need a separate variable.